client/src/components/Monument/Update/MonumentUpdate.js

```diff
@@ -0,0 +1,430 @@
+import * as React from 'react';
+import './MonumentUpdate.scss';
+import AttributeChange from './AttributeChange/AttributeChange';
+import { prettyPrintDate, prettyPrintDateString } from '../../../utils/string-util';
+import UnchangedTags from './TagChanges/UnchangedTags/UnchangedTags';
+import AddedTags from './TagChanges/AddedTags/AddedTags';
+import RemovedTags from './TagChanges/RemovedTags/RemovedTags';
+import ReferenceChanges from './References/ReferenceChanges/ReferenceChanges';
+import AddedReferences from './References/AddedReferences/AddedReferences';
+import DeletedReferences from './References/DeletedReferences/DeletedReferences';
+import AddedImages from './Images/AddedImages/AddedImages';
+import DeletedImages from './Images/DeletedImages/DeletedImages';
+import { Collapse } from 'react-bootstrap';
+
+/**
+ * Presentational component for displaying changes to a Monument
+ */
+export default class MonumentUpdate extends React.Component {
```

Comment:
Most of the logic in this class was in `UpdateReviewModal` before. I pulled it out into this component so I could reuse it for displaying `UpdateMonumentSuggestion`s.

Reply:
We're going to have a lot of conflicts on the 360 images branch :cry:

Reply:
😭